frontend/common/code-help/traits/traits-flutter.js

```diff
@@ -1,16 +1,28 @@
-module.exports = (envId, { LIB_NAME, USER_ID, LIB_NAME_JAVA, TRAIT_NAME, FEATURE_NAME, FEATURE_FUNCTION, FEATURE_NAME_ALT, FEATURE_NAME_ALT_VALUE, NPM_CLIENT }, userId) => `final user = FeatureUser(identifier: '${USER_ID}');
+module.exports = (envId, { LIB_NAME, USER_ID, LIB_NAME_JAVA, TRAIT_NAME, FEATURE_NAME, FEATURE_FUNCTION, FEATURE_NAME_ALT, FEATURE_NAME_ALT_VALUE, NPM_CLIENT }, userId) => `final flagsmithClient = FlagsmithClient(
+    apiKey: '${envId}' 
+    config: config, 
+    seeds: <Flag>[
+        Flag.seed('feature', enabled: true),
+    ],
+);
 
-final userTrait = await flagsmithClient.getTrait(user, '${TRAIT_NAME}');
-if (userTrait != null) {
-    // run the code to use user trait
-} else {
-    // run the code without user trait
-}
+//if you prefer async initialization then you should use
+//final flagsmithClient = await FlagsmithClient.init(
+//        apiKey: 'YOUR_ENV_API_KEY',
+//        config: config, 
+//        seeds: <Flag>[
+//            Flag.seed('feature', enabled: true),
+//        ], 
+//        update: false,
+//    );
 
-// Or get specific traits 
-final userTraits = await flagsmithClient.getTraits(user, keys: ['${TRAIT_NAME}', 'other_trait']);
+// This will create a user in the dashboard if they don't already exist
```

Comment:
Pretty sure just creating an object in the client won't interact with the API will it? think we should be clear about what point exactly the user is created - this seems a bit misleading to me.